_drafts/bridgetownrb.md

```diff
@@ -0,0 +1,31 @@
+# Day 16 - `bridgetown` gem - Ruby-powered static site generation in 2020
+
+Written by [Jared White](https://twitter.com/jaredcwhite) from [Whitefusion](https://whitefusion.io/)  {% avatar jaredcwhite %}
```

Comment:
This was intended to be the spot for the author of *the writeup* so put your name here :)

Reply:
No worries. I will change it before going live and add a little bio if I can find something on your website or twitter or such :-).  Thanks again.

Reply:
Task failed successfully from my side, I reckon :D let me fix both my submissions accordingly in the upcoming days!